packages/expo-test-runner/package.json

```diff
@@ -28,7 +28,6 @@
   },
   "devDependencies": {
     "@babel/core": "^7.9.0",
-    "@tsconfig/node14": "^1.0.3",
```

Comment:
Think we want to keep the tsconfig preset as a dev dep since expo-test-runner's own tsconfig.json specifies `@tsconfig/node18`